src/quo2/components/code/snippet_preview/view.cljs

```diff
@@ -0,0 +1,86 @@
+(ns quo2.components.code.snippet-preview.view
```

Comment:
This namespace is repeating a lot of code from `quo2.components.code.snippet`, IMO is better to generalize `quo2.components.code.snippet` or have a common helper component for both so we can have a single implementation. Otherwise, any change in the appearance or bug solved will need to be carefully propagated and it may bring more bugs hard to track.

Reply:
@ulisesmac I've been thinking of it, but not found the best practice. Do you have any idea?